specs/MultiProfile.md

```diff
@@ -107,6 +108,9 @@ HRESULT AppWindow::OnCreateCoreWebView2ControllerCompleted(HRESULT result, ICore
         BOOL inPrivateModeEnabled = FALSE;
         CHECK_FAILURE(profile->get_IsInPrivateModeEnabled(&inPrivateModeEnabled));
         CHECK_FAILURE(profile->get_ProfileName(&m_profileName));
+
+        // Gets the cookieManager which can be used the same as described in CookieManagement.md
+        CHECK_FAILURE(profile->get_CookieManager(&m_cookieManager));
```

Comment:
You need to add new samples. This sample is using the profile name and InPrivate status to update the title and icon of the window. The CookieManager isn't going to help with updating the title and icon of the window. For a new sample you may want to look at the CookieManager API spec and adapt its sample code.

Reply:
Added new samples.